code-challenges.test.js

```diff
@@ -64,4 +113,31 @@ const accountTransactions2 = [250, -89, 100, -96]
 const accountTransactions3 = []
 // Expected output: []
 
+// describe("trans", () => {
+//   it("akes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+//     expect(fiboSeq(accountTransactions1)).toEqual("[100, 83, 60, 51]")
+//     expect(fiboSeq(accountTransactions2)).toEqual("[250, 161, 261, 165]")
+//     expect(fiboSeq(accountTransactions3)).toEqual("[]")
+
+
+//   })
+// })
+
 // b) Create the function that makes the test pass.
+
+// pseudo code:
+// input: takes an array
+// output: the array's accumulated sum as it goes into the next index
+// create a function "trans" that takes an array as a function
+// declared where the index iteration will start
+// using .map() on the arry bring the sum of the values iterated by the next index. 
+
```

Comment:
Do we need to declare where the iteration will start with .map? Or are we using i = 0 as the initial value?

Reply:
we're using i = 0 as the initial value for the index and to start counting from there.